sda/service.go

```diff
@@ -359,43 +375,34 @@ func CreateServiceMessage(service string, r interface{}) (*InterServiceMessage,
 
 }
 
-// Client is a simple client structure to be used when wanting to connect to services. It
-// holds the private and public key and allows to connect to a service through
-// the network.
-// The error-handling is done using the ErrorRet structure which can be returned
-// in place of the standard reply. The Client.Send method will catch that and return
-// the appropriate error.
+// Client is a struct used to communicate with a remote Service running on a
+// sda.Conode
 type Client struct {
```

Comment:
`network.Client` handles the *network operations*: it connects to an node, sends the message and waits for a response.
`sda.Client` handles the marshalling of a message into a `sda.ClientRequest` with the *right ServiceID*, pass it to the network, and also some convenience methods to send to all nodes.